src/pool/srv_pool.c

```diff
@@ -5904,21 +5954,21 @@ ds_pool_tgt_exclude_out(uuid_t pool_uuid, struct pool_target_id_list *list)
 {
 	return pool_update_map_internal(pool_uuid, POOL_EXCLUDE_OUT, false,
 					list, NULL, NULL, NULL, NULL, NULL,
-					NULL);
+					NULL, false);
```

Comment:
Hmm, even if the rank/tgt is excluded by dmg, then following rebuild process will call this function to set the target status to "DOWNOUT",  so it should tell if the "exclude out" is from dmg or swim?

Reply:
I I don't think it's necessary. Why should we distinguish